command/agent/template/template.go

```diff
@@ -25,6 +28,15 @@ type ServerConfig struct {
 	ExitAfterAuth bool
 
 	Namespace string
+
+	// LogLevel is needed to set the internal Consul Template Runner's log level
+	// to match the log level of Vault Agent. The internal Runner creates it's own
+	// logger and can't be set externally or copied from the Template Server.
+	//
+	// LogWriter is needed to initialize Consul Template's internal logger to use
+	// the same io.Writer that Vault Agent itself is using.
+	LogLevel  log.Level
```

Comment:
Using `hclog` instead of `log` here and in `logLevelToStringPtr` will avoid a second import of `go-hclog`.

Reply:
Great catch, thanks! I rebased and cleaned up the PR in 52272b7c4